ckanext/switzerland/tests/test_format_utils.py

```diff
@@ -17,29 +16,14 @@
     import unittest
 
 class TestPlugin(unittest.TestCase):
```

Comment:
The setup of the test is simplified now, because the format mapping is part of the loaded module ``ckanext.switzerland.helpers.format_utils`` and there is no need for a separate ``test_mapping.yaml`` any more. The real ``m̀apping.yaml`` will be loaded directly with the module.